### PR TITLE
Remove unnecessary `#` hash characters

### LINKED ISSUE #354: 2.1.1 Experiment with Simulators in Grover's has an unnecessary # in code
### Steps to reproduce the problem

1. Go to 2.1.1 Experiment with Simulators 
2. Click on run in code block
3. Scroll down to results

### What is the current behavior?

It returns an error 

### What is the expected behavior?

It should return the statevector 

### Screenshots (optional but very helpful)
Screen Shot 2022-01-27 at 8 22 04 AM

If applicable, please add screenshots or video captures to help explain the problem.

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Fixes #354 

## Implementation details

Removed characters that were commenting out code

## How to read this PR

- Review extra hash characters that were removed
- try to visit the the [Grovers Algorithm chapter](https://platypus-pr-898.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/ch-algorithms/grovers-algorithm) in the preview branch
- run the code in the 2.1.1 cell and see the error is gone

## Screenshots

Screen Shot 2022-04-12 at 9 54 48 AM

## Note
This is my first attempt at fixing one of these code cells; I based the removal of the `#` characters on how I see the `vector2latex` used in other notebooks... hopefully this is right.

### CODE REVIEW

File: notebooks/ch-algorithms/grover.ipynb [hunk 1]
```diff
@@ -1286,8 +1286,8 @@
     "result = sv_sim.run(grover_circuit).result()\n",
     "statevec = result.get_statevector()\n",
     "statevec\n",
-    "#from qiskit_textbook.tools import vector2latex\n",
-    "#vector2latex(statevec, pretext=\"|\\\\psi\\\\rangle =\")"
+    "from qiskit_textbook.tools import vector2latex\n",
```
Comment (vabarbosa) [inline notebooks/ch-algorithms/grover.ipynb:1289]: (just  a preference thing but) should this `import` statement be moved to the top of the cell/code block?

**Review by vabarbosa — COMMENTED** [review]
@techtolentino thank you! i just had one minor comment, but we can let @frankharkins weigh in on if this can be merged

**Review by frankharkins — CHANGES_REQUESTED** [review]
Thanks Randy! This function actually [made it into Qiskit](https://qiskit.org/documentation/stubs/qiskit.visualization.array_to_latex.html) now, so you can do
```
from qiskit.visualization import array_to_latex
array_to_latex(statevec, prefix="...")
```

File: notebooks/ch-algorithms/grover.ipynb [hunk 2]
```diff
@@ -1286,8 +1286,8 @@
     "result = sv_sim.run(grover_circuit).result()\n",
     "statevec = result.get_statevector()\n",
     "statevec\n",
```
Comment (frankharkins) [inline notebooks/ch-algorithms/grover.ipynb:1288]:
```suggestion
```
This has no effect now

File: notebooks/ch-algorithms/grover.ipynb (hunk 1 above)
Comment (frankharkins) [reply notebooks/ch-algorithms/grover.ipynb:1289]: No strong preference from me; I think it's slightly easier to read the cell this way, i.e. "simulate, import fn, use fn" vs "import fn, simulate, use fn", but it's maybe more common to keep imports at the top.

Comment (vabarbosa) [reply notebooks/ch-algorithms/grover.ipynb:1288]: no problem. it can remain as is!

**Review by frankharkins — APPROVED** [review]
LGTM, thanks!

**Review by vabarbosa — APPROVED** [review]
thanks @techtolentino

**Pull request merged**